### DISCUSSION TITLE
How change color to ";" and whole text after?

### DISCUSSION THREAD

**Discussion by Alexnoj** [mark-wiemer/ahkpp#416] [General]
How change color to ";" and whole text after?
Thanks
![image](https://github.com/mark-wiemer-org/ahkpp/assets/30004592/f2554cb7-2650-4db9-8759-4269c5b1e457)

**Comment by mark-wiemer** [comment]
Hi, a semi-colon and anything after it is usually coded as a "comment" because that's how AHK treats it. This allows comments in AHK to appear the same as other programming languages in that editor. The actual color you see will determine on your VS Code color theme, which you can change via F1 > "Color Theme". Let me know if you have other questions though!
